wandb/cli.py

```diff
@@ -52,41 +52,25 @@ def editor(content='', marker='# Enter a description, markdown is allowed!\n'):
 # returned by api.settings()
 CONTEXT=dict(default_map=api.settings())
 
-class BucketGroup(click.Group):
+class RunGroup(click.Group):
     @display_error
     def get_command(self, ctx, cmd_name):
+        #TODO: check if cmd_name is a file in the current dir and not require `run`?
```

Comment:
I was thinking we could make `wandb` without a sub command run a file pretty easily here.